packages/react-theme/src/utils/themeToCSSVariables.ts

```diff
@@ -0,0 +1,25 @@
+import { Theme } from '../types';
+
+function flattenThemeToCSSVariables(
```

Comment:
seems to make more sense to me to have these util functions as a part of the provider package